Model/VatNumber/Config/Converter.php

```diff
@@ -31,35 +31,40 @@ public function __construct(BooleanUtils $booleanUtils)
     }
 
     /**
-     * Convert config.
+     * Convert config creating an assoc array with the country code as key
+     * and the pattern and other properties as value
      *
      * @param DOMDocument $source
      * @return array
+     * @throws \InvalidArgumentException
      */
     public function convert($source): array
     {
         $result = [];
 
-        /** @var DOMNode $vatNumberNode */
+        /** @var \DOMNode $vatNumberNode */
```

Comment:
The import becomes obsolete with this change, right?

Reply:
Removed use